packages/eas-cli/src/commandUtils/EasCommand.ts

```diff
@@ -203,6 +203,8 @@ export default abstract class EasCommand extends Command {
       Log.error(err.message);
     }
     Log.debug(err);
```

Comment:
When you run with EXPO_DEBUG=1 this log will be printed and it will contain a real stacktrace.

I think the problem here is more caused by the fact that we have to independent debug varaible, so may be instead of that fix we could make debug logs to be printed when either EXPO_DEBUG=1 or DEBUG=* is set

Reply:
ah good point @wkozyra95! agreed that it is confusing that we have both env vars for debugging purposes here. in either case i think that it is still not a bad idea to ensure the rethrown error includes the original stack